airbyte-integrations/connectors/source-facebook-marketing-api-singer/README.md

```diff
@@ -0,0 +1,66 @@
+```
+If you are in an IDE, follow your IDE's instructions to activate the virtualenv. 
+
+**All the instructions below assume you have correctly activated the virtualenv.**.
+
+### Locally running the connector
+```
+python main_dev.py spec
+python main_dev.py check --config sample_files/test_config.json
+python main_dev.py discover --config sample_files/test_config.json
+python main_dev.py read --config sample_files/test_config.json --catalog sample_files/test_catalog.json
+```
+
+### Unit Tests
+To run unit tests locally, from the connector root run:
+```
+pytest unit_tests
+```
+
+
+### Locally running the connector docker image
+Follow the instructions in the "Configure Credentials" section to generate the connector config in `secrets/config.json`, then run the following 
+commands from the airbyte root directory. 
+
+```
+# in airbyte root directory
+./gradlew :airbyte-integrations:connectors:source-facebook-marketing-api-singer:buildImage
+docker run --rm airbyte/source-facebook-marketing-api-singer:dev spec
+docker run --rm -v $(pwd)/airbyte-integrations/connectors/source-facebook-marketing-api-singer/secrets:/secrets airbyte/source-facebook-marketing-api-singer:dev check --config /secrets/config.json
+docker run --rm -v $(pwd)/airbyte-integrations/connectors/source-facebook-marketing-api-singer/secrets:/secrets airbyte/source-facebook-marketing-api-singer:dev discover --config /secrets/config.json
+docker run --rm -v $(pwd)/airbyte-integrations/connectors/source-facebook-marketing-api-singer/secrets:/secrets -v $(pwd)/airbyte-integrations/connectors/source-facebook-marketing-api-singer/sample_files:/sample_files airbyte/source-facebook-marketing-api-singer:dev read --config /secrets/config.json --catalog /sample_files/sample_catalog.json
+```
+
+### Integration Tests 
+1. Configure credentials as appropriate, described below.
+1. From the airbyte project root, run `./gradlew :airbyte-integrations:connectors:source-facebook-marketing-api-singer:standardSourceTestPython` to run the standard integration test suite.
+1. To run additional integration tests, place your integration tests in the `integration_tests` directory and run them with `pytest integration_tests`.
+   Make sure to familiarize yourself with [pytest test discovery](https://docs.pytest.org/en/latest/goodpractices.html#test-discovery) to know how your test files and methods should be named.
+
+## Dependency Management
+All dependencies should go in `setup.py`, NOT `requirements.txt`. The requirements file is only used to connect internal Airbyte dependencies in the monorepo for local development.
+
+## Configure credentials
+### Configuring credentials as a community contributor
+Follow the instructions in the [documentation](https://docs.airbyte.io/integrations/sources/facebook-marketing-api) to generate an access token. 
+Then create a file `secrets/config.json` conforming to the `spec.json` file. See `sample_files/sample_config.json` for a sample config file.
```

Comment:
if we are creating `screts/config.json` shouldn't that be what the docker command line arguments refer to in your sample above?